### PR TITLE
Upgrade web-ext and update .gitignore

### PR DESCRIPTION
**Pull request by rehandalal**
Seems to fix styling for me.

r?

### CODE REVIEW

**Comment by mythmon** [comment]
Odd. This didn't fix the problem for me. I ran

```
rm -rf node_modules
yarn install
yarn build
```

and then installed `web-ext-artifacts/normandy_devtools.xpi` as a temporary add-on in Firefox Nightly 77 with `extensions.legacy.enabled` and `extensions.experiments.enabled` set to true.

I still see the lack of styling.

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,5 @@ dist/
 node_modules/
 web-ext-artifacts/
 yarn-error.log
+.webpack-cache/
```
Comment (mythmon) [inline .gitignore:5]: `.webpack-cache` doesn't exist on this branch. I don't think it needs to be in `.gitignore.

Comment (rehandalal) [reply .gitignore:5]: i put this there because switching between branches becomes annoying

**Comment by rehandalal** [comment]
Closing in favor of simplified 1.x branch

**Pull request closed without merging**